conf/cassandra.yaml

```diff
@@ -784,6 +784,15 @@ compaction_throughput_mb_per_sec: 16
 # between the sstables, reducing page cache churn and keeping hot rows hot
 sstable_preemptive_open_interval_in_mb: 50
 
+# When enabled, permits Cassandra to zero copy eligible SSTables between
```

Comment:
Worth documenting this optimization relies on having sstables that are wholly contained in the range being streamed and that configurations like size tiered without 6696 won't benefit, and would in fact spend time reading indexes checking to see if the optimization applies? Leveled should always benefit from this optimization.

Reply:
3 node cluster with RF=3?

Reply:
See above.

Reply:
Updated.